frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx

```diff
@@ -539,8 +539,7 @@ class ExpressionEditorTextfield extends React.Component<
     },
     {
       name: "chooseSuggestion",
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore // Based on typings null is not a valid value, but bindKey is assigned dynamically if there are suggestions available.
+      // @ts-expect-error Based on typings null is not a valid value, but bindKey is assigned dynamically if there are suggestions available.
```

Comment:
Suppressing TS here is a hack (types are inaccurate), not an error that we expect.